Fix csv examples (#3345)
influxdata/docs-v2: content/influxdb/v2.1/write-data/developer-tools/csv.md

```

{{% note %}}
#### Required annotations and columns

> To write CSV data to InfluxDB with Flux, _all_ of the following annotations and columns must be present.
> Required annotations:

Reviewer: ```suggestion
To write CSV data to InfluxDB with Flux, you must include _all_ of the following annotations and columns:
```

- `datatype`
- `group`
- `default`

See [annotations](/influxdb/v2.1/reference/syntax/annotated-csv/#annotations) for more information.

> With Flux, there must also be a comma between the annotation name and the annotation values (this differs from the `influx write` command).

Reviewer: ```suggestion
With Flux, you must also include a comma between the annotation name and the annotation values (this differs from the `influx write` command).
```

> An example of valid Flux syntax can be found [here](/influxdb/v2.1/reference/syntax/annotated-csv/#annotated-csv-in-flux).

Reviewer: ```suggestion
See an example of valid syntax for [annotated CSV in Flux](/influxdb/v2.1/reference/syntax/annotated-csv/#annotated-csv-in-flux).
```
